pygmt/clib/session.py

```diff
@@ -1641,7 +1641,7 @@ def extract_region(self):
         0.00, 10.00, -20.00, -10.00
 
         Using ISO country codes for the regions (for example ``'US.HI'`` for
```

Comment:
It's only a minor detail, but for consistency, we also should use double quotation marks in comments.
Please also update line 1657.
```suggestion
        Using ISO country codes for the regions (for example ``"US.HI"`` for
```

Reply:
Done at 8c57ea4148dc70cc59073329b0ca2a584b9af48c.

Reply:
Thanks!